_data/songs.json

```diff
@@ -7,10 +7,31 @@
     "detail": "สมเด็จพระเทพรัตนราชสุดา ฯ สยามบรมราชกุมารี ทรงพระราชนิพนธ์เพลง \"ส้มตำ\" ซึ่งเป็นเพลงไทยลูกทุ่ง เมื่อ พ.ศ. ๒๕๑๓"
   },
   {
-    "url": "",
-    "title": "",
-    "melody": "",
-    "lyric": "",
-    "detail": ""
+    "url": "https://www.youtube.com/watch?v=FjjQoZ4wDno",
```

Comment:
One small note: we need to reference the video from youtube's embed link to display properly. It should be `https://www.youtube.com/embed/FjjQoZ4wDno`.